packages/react-query/src/__tests__/useQuery.test.tsx

```diff
@@ -1201,96 +1131,6 @@ describe('useQuery', () => {
     expect(states[1]).toMatchObject({ data: 'test' })
   })
 
-  it('should be able to remove a query', async () => {
-    const key = queryKey()
-    const states: UseQueryResult<number>[] = []
-    let count = 0
-
-    function Page() {
-      const [, rerender] = React.useState({})
-      const state = useQuery(key, () => ++count, { notifyOnChangeProps: 'all' })
-
-      states.push(state)
-
-      const { remove } = state
-
-      return (
-        <div>
-          <button onClick={() => remove()}>remove</button>
-          <button onClick={() => rerender({})}>rerender</button>
-          data: {state.data ?? 'null'}
-        </div>
-      )
-    }
-
-    const rendered = renderWithClient(queryClient, <Page />)
-
-    await waitFor(() => rendered.getByText('data: 1'))
-    fireEvent.click(rendered.getByRole('button', { name: /remove/i }))
-
-    await sleep(20)
-    fireEvent.click(rendered.getByRole('button', { name: /rerender/i }))
-    await waitFor(() => rendered.getByText('data: 2'))
-
-    expect(states.length).toBe(4)
-    // Initial
-    expect(states[0]).toMatchObject({ status: 'loading', data: undefined })
-    // Fetched
-    expect(states[1]).toMatchObject({ status: 'success', data: 1 })
-    // Remove + Hook state update, batched
-    expect(states[2]).toMatchObject({ status: 'loading', data: undefined })
-    // Fetched
-    expect(states[3]).toMatchObject({ status: 'success', data: 2 })
-  })
-
-  it('should create a new query when refetching a removed query', async () => {
-    const key = queryKey()
-    const states: UseQueryResult<number>[] = []
-    let count = 0
-
-    function Page() {
-      const state = useQuery(
-        key,
-        async () => {
-          await sleep(10)
-          return ++count
-        },
-        { notifyOnChangeProps: 'all' },
-      )
-
-      states.push(state)
-
-      const { remove, refetch } = state
-
-      return (
-        <div>
-          <button onClick={() => remove()}>remove</button>
```

Comment:
we could also keep this test and just do `queryClient.removeQueries({ queryKey: key })` here

Reply:
Done ✅